util/hint/hint_processor.go

```diff
@@ -28,6 +28,13 @@ import (
 	"go.uber.org/zap"
 )
 
+var supportedHintNameForInsertStmt = map[string]struct{}{}
+
+func init() {
+	supportedHintNameForInsertStmt["MEMORY_QUOTA"] = struct{}{}
+	supportedHintNameForInsertStmt["memory_quota"] = struct{}{}
```

Comment:
Please maintain only one map entry for one hint. BTW, can we support all hints for `INSERT` statements?

Reply:
updated.  And for this question `can we support all hints for INSERT statements` , I suggest discussing this in another issue.

Reply:
which issue?

Reply:
https://github.com/pingcap/tidb/issues/18049 I think we can discuss the ranges of hints we are going to support for `Insert` in this issue. @zz-jason